### ISSUE TITLE
Extension not working

### ISSUE THREAD

**Issue by Gliptal** [sketchbuch/vsc-workspace-sidebar#10]
Doesn't seem to be working for me on VSCode 1.52.0 (System Setup), Windows 10 20H2 (19042.630).

`workspaceSidebar.folder` is correctly set to a folder that has multiple `.code-workspace` files in its subfolders, all within 5 subdirectories. `"workspaceSidebar.depth"` is also set to 5.

**Comment by sketchbuch** [comment]
Thanks for reporting. I don't use Windows mself so never really checked. I shall be checking soon to get the extension working on windows.

**Comment by sketchbuch** [comment]
Are you sure the path is correct? What is the path?

It worked in my Windows VM:

![vsc-win](https://user-images.githubusercontent.com/16445037/103020234-57ec5500-4548-11eb-87c3-dffe7d8c7350.png)

**Comment by sketchbuch** [comment]
The icon for the active Workspace is not showing - I shall look into this. The extension itself though seems to be working.

**Comment by sketchbuch** [comment]
@Gliptal Please try v1.2.0 - I have tested the extension now in a Windows VM and it is working.

**Comment by Gliptal** [comment]
Unfortunately it still doesn't seem to be working for me.

**Comment by sketchbuch** [comment]
What is the path? Can you post it here? I shall have another look. Maybe it is specific to do with the way you wrote the path.

The folder it points to is a real folder? Or is it some kind of symbolic link? Junction?

**Comment by Gliptal** [comment]
Here are my settings, no symbolic links or junctions of any sort. 

The redacted text is made of basic alphabet characters (no accents or similar); there is a dot `.` in the path though. I have a number of `.code-workspace` files within 5 folders of the set parent folder, placed in a hidden `.vscode` directory. I tried unhiding it to no avail.

![Code_AYZED50f6d](https://user-images.githubusercontent.com/3907646/109132936-b4404080-7754-11eb-9137-669b297f7fee.png)

**Comment by sketchbuch** [comment]
Hmmm. maybe the dot is the issue. I shall have another look into this because I tested paths similar to yours in my VM and it was working

**Comment by Gliptal** [comment]
Is there a log I can send you to help out?

**Comment by sketchbuch** [comment]
Not at the moment. I'll look into this and get back to you...

**Comment by sketchbuch** [comment]
Hi,

I have just published a new version: 1.2.5. This should fix the issues, the problem was caused by the workspaces being in a hidden folder. I was excluding hiddne folders, now I have a list of allowed folders that are checked first, I added .vscode to this and it is working.

The dot in your username had nothing to do with it, I also tested this.

I also discovered vcode has inconsistent drive letter casing on windows, so I now lowercase workspace file paths when cheking for an active workspace on windows. Thus the highlighting on windows should now work.
